client/cl_main.lua

```diff
@@ -172,12 +191,23 @@ function CameraLoop()
         RenderScriptCams(true, false, 0, true, false)
 
         while camera and not IsEntityDead(lPed) and (GetVehiclePedIsIn(lPed) == vehicle) do
+            if doFlash then
+                SendNUIMessage({action = "toggleFlash", status = '<b style="color:green;">ON</b>'})
+            else
+                SendNUIMessage({action = "toggleFlash", status = '<b style="color:red;">OFF</b>'})
+            end
```

Comment:
Refactor this code. It has cyber security concerns since people can intercept the status which gets inserted raw into the DOM, making it possible for hacking to inject whatever code they want into the user's UI.

Reply:
Definitely, i will rework on this code.